lib/fill/index.js

```diff
@@ -0,0 +1,9 @@
+import curryN from '../curryN'
+
+export default curryN(2, function fill(arr, value) {
```

Comment:
[`_.fill`](https://lodash.com/docs/4.17.4#fill) also has `start` and `end` args to fill only specified range  
Leave `curryN(2)`, if you'll add `start` and `end`

Reply:
Sorry, do not understand what do you mean. We need to return back _start_ and _end_ args, but leave curryN(2)?

Reply:
`curryN(2, function fill(arr, value, start, end) {})`
`curryN(2)` allows use `fill(arr)(value)` without start/end + use `fill(arr)(value, start)` to use without end + use `fill(arr)(value, start, end)` to use without end  
But I think the better idea will be:
`fill(val)(arr)`
`fill(val, start)(arr)`
`fill(val, start, end)(arr)`
Still needs `curryN(2)` but also needs to check where is arr (just with `arguments.length`)